### DISCUSSION TITLE
Sorting the playlist during playback is not possible.

### DISCUSSION THREAD

**Discussion by wellro1** [dtcooper/raspotify#524] [Q&A] [answered]
Hello all,
I am using a Raspberry 3B in combination with a Hifiberry Dac Plus running Raspotify. Everything is working fine. Except for one thing. If I play a playlist on Spotify, I can re-sort it on my phone and on all other playback devices (e.g. Teufel 3Sixty) but not on Raspotyfy. The playlist is then greyed out and the tracks are not pickable. I have tried different phones with different operating systems, without success. Also different Spotify accounts always led to the same result. The playlist remains greyed out during playback.
Is this generally not possible or did I configure something wrong in the settings, maybe under /etc/raspotify/conf?

**Comment by JasonLG1979** [comment, marked as answer]
That is a know upstream issue.
